data/companies.yml

```diff
@@ -667,6 +668,12 @@ certification:
     details:
       - "The Istio Certified Associate (ICA) exam demonstrates a solid understanding of Istio principles, terminology, and best practices for setting up Istio."
       - "The ICA is a pre-professional certification designed for engineers, CI/CD practitioners, or anyone with special interests in Istio. A certified ICA will confirm their foundational knowledge of Istio principles, terminology, and best practices and demonstrate their ability to set up Istio."
+  - name: "IT-Schulungen.com"
```

Comment:
I think what you had to start with is more correct: you're not offering a different certification, you're offering training for the CNCF's ICA cert.